### DISCUSSION TITLE
Management rights on challenge & opportunity level

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#392] [Q&A] [answered]
I just found out that only as a global admin I can use functions such as:
- add a project
- add solution details
- add an actor group

I should also be able to do this as ecoverse/challenge admin, right?

**Comment by Comoque1** [comment, marked as answer]
Thanks for the feedback, Flori! The [bug](https://app.zenhub.com/workspaces/alkemio-5ecb98b262ebd9f4aec4194c/issues/alkem-io/client-web/917) is as added in the backlog.
